packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap

```diff
@@ -805,83 +830,44 @@ exports[`Media Player: Placeholder should render an audio placeholder without du
     </button>
     <img
       alt=""
-      class="emotion-13 emotion-14"
+      class="emotion-15 emotion-16"
       src="http://foo.bar/placeholder.png"
     />
   </div>
 </div>
 `;
 
 exports[`Media Player: Placeholder should render no-js styles when noJsClassName prop is used 1`] = `
-.emotion-6 {
-  -webkit-clip-path: inset(100%);
-  clip-path: inset(100%);
-  -webkit-clip: rect(1px,1px,1px,1px);
-  clip: rect(1px,1px,1px,1px);
-  height: 1px;
-  overflow: hidden;
+.emotion-0 {
   position: absolute;
-  width: 1px;
-  margin: 0;
-}
-
-.emotion-10 > svg {
-  color: #FFFFFF;
-  fill: currentColor;
-  height: 1.5rem;
-  margin-top: 0.5rem;
-  width: 1.5rem;
-}
-
-.emotion-8 {
-  vertical-align: middle;
-  margin: 0 0.25rem;
-  color: #222222;
-  fill: currentColor;
-  width: 0.75rem;
-  height: 0.75rem;
-}
-
-.emotion-12 {
-  display: block;
-  margin-top: 0.5rem;
-}
-
-.emotion-17 {
-  display: block;
+  top: 0;
+  left: 0;
   width: 100%;
-  visibility: visible;
+  height: 100%;
+  cursor: default;
 }
 
-.emotion-0 {
-  display: block;
-  font-weight: normal;
-  padding: 0.5rem;
-  border-bottom: 0.0625rem solid transparent;
+.emotion-1:hover .css-9vh530-StyledPlayButton,
+.emotion-1:focus .css-9vh530-StyledPlayButton {
+  background-color: #B80000;
 }
 
-@media screen and (-ms-high-contrast:active) {
-  .emotion-0 {
-    background-color: window;
-  }
+.emotion-1:hover .css-1uhr2ah-Button-StyledPlayButton,
+.emotion-1:focus .css-1uhr2ah-Button-StyledPlayButton {
+  background-color: #B80000;
 }
 
-@media (min-width:25rem) {
-  .emotion-0 {
-    padding: 1rem;
-  }
+.emotion-1:hover .css-pz3gmj-StyledPlayButton,
+.emotion-1:focus .css-pz3gmj-StyledPlayButton {
+  background-color: #B80000;
 }
```

Comment:
These are unused quadruplicates of the styles below

Accounts for 15/40 of the new lines

Reply:
Closing all the placeholder snapshot comments - a fix to the styles achieved snapshot parity 👌